bin/spark-shell

```diff
@@ -32,7 +32,10 @@ if [ -z "${SPARK_HOME}" ]; then
   source "$(dirname "$0")"/find-spark-home
 fi
 
-export _SPARK_CMD_USAGE="Usage: ./bin/spark-shell [options]"
+export _SPARK_CMD_USAGE="Usage: ./bin/spark-shell [options]
+
+Scala REPL options:
+  -I <file>                   preload <file>, enforcing line-by-line interpretation"
```

Comment:
where do we define other options?

Reply:
I tested other options and this one looks only the valid one. I described in PR description.

Reply:
I mean, I didn't find
```
Options:
  --master MASTER_URL         spark://host:port, mesos://host:port, yarn,
                              k8s://https://host:port, or local (Default: local[*]).
  --deploy-mode DEPLOY_MODE   Whether to launch the driver program locally ("client") or
                              on one of the worker machines inside the cluster ("cluster")
                              (Default: client).
```

in the shell script. Where do we define them?

Reply:
Oh haha. Sorry. That's in `SparkSubmitArguments.printUsageAndExit`.thats why I left https://github.com/apache/spark/pull/22919#discussion_r230554455

Reply:
Shall we also define `-i` behavior here? I think for now this option is also accepted by the REPL.

Reply:
Yea .. but `-i` doesn't handle implicits like toDF or symbols which are pretty basic ones. I think we should better avoid to document it for now.